### PR TITLE
STABLE-9: OXT-1549: [installer] Only install upgradecompat on upgrades

### PR DESCRIPTION
**Pull request by crogers1**
OXT-1549

Signed-off-by: Chris <rogersc@ainfosec.com>

### CODE REVIEW

File: part2/stages/Functions/install-main [hunk 1]
```diff
@@ -331,20 +331,21 @@ install_dom0()
...
+
+		if [ "$(uname -m)" = "i686" ]; then
+			do_cmd create_lv "xenclient" "upgradecompat" \
+				--size "${COMPAT_LV_SIZE}" >&2 || return 1
+				write_rootfs "${UPGRADE_ROOTFS}" "${ROOTFS_TYPE}" /dev/xenclient/upgradecompat >&2 || {
+					do_cmd lvremove -f /dev/xenclient/upgradecompat >&2
+						return 1
+				}
+		fi
+	fi
```
Comment (eric-ch) [inline part2/stages/Functions/install-main]: Indentation changed from 4-spaces to tabs.

Comment (crogers1) [reply part2/stages/Functions/install-main]: uh oh.  Will fix

**Comment by eric-ch** [comment]
Building: [$6531](http://openxt-builder.ainfosec.com:8010/builders/openxt64/builds/6531)

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt64/builds/6550

**Comment by jean-edouard** [comment]
LGTM, the file is still downloaded and verified (expected since it's in the manifest), but not extracted or installed.
Merging soon.

**Pull request merged**